src/helper/factory/index.test.ts

```diff
@@ -388,9 +388,7 @@ describe('createFactory', () => {
 
       const routes = new Hono().use('*', middleware)
       type Actual = ExtractSchema<typeof routes>
-      type Expected = {
-        '*': {}
-      }
+      type Expected = {}
```

Comment:
Though you change the test, I also think it's okay. It's not a problem for real-world use cases. Makes sense for me 👍